public/app/features/auth-config/AuthProvidersListPage.tsx

```diff
@@ -120,6 +120,12 @@ export const AuthConfigPageUnconnected = ({
                   configPath={settings.configPath}
                 />
               ))}
+            {config.buildInfo.edition === GrafanaEdition.OpenSource && (
```

Comment:
Would this also appear in un-licensed Grafana Enterprise?

Reply:
@mjseaman no, only in the open source build